canary/apps/react/cra-ts/package.json

```diff
@@ -19,10 +19,11 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "dev": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test",
+    "start": "serve build -s -p 3000",
+    "eject": "react-app-rewired eject"
```

Comment:
I'm using this `react-app-rewired` library so we can import files outside of src folder. This only effects CRA an CRA-TS